terraform/modules/data-share-service/lb.tf

```diff
@@ -33,6 +33,24 @@ resource "aws_lb_listener" "listener-http" {
   depends_on = [aws_lb_target_group.green]
 }
 
+#tfsec:ignore:aws-elb-http-not-used
+resource "aws_lb_listener" "test-listener-http" {
```

Comment:
Terraform encourages naming with _ as separators, but not too worried

Reply:
doh! will fix of course